core/org.wso2.carbon.user.core/src/main/java/org/wso2/carbon/user/core/authorization/JDBCAuthorizationManager.java

```diff
@@ -1010,6 +1013,32 @@ public int getTenantId() throws UserStoreException {
         return tenantId;
     }
 
+    /**
+     * @param name
+     * @return
+     */
+    private String modify(String name) {
```

Comment:
It it better to have more expressive method names. It will help other developers to easily understand the behavior of a method.
Avoid using generic method names like "modify".